lib/test.js

```diff
@@ -88,6 +89,20 @@ function Test (options) {
   Readable.apply(this, options)
 }
 
+Test.prototype.beforeEach = function (fn) {
+  this.on('beforechild', fn)
```

Comment:
That's kinda what I thought. I think it would be reasonable to do it outside of node-tap.